content/docs/add-react-to-a-website.md

```diff
@@ -1,202 +1,203 @@
 ---
 id: add-react-to-a-website
-title: Add React to a Website
+title: Dodaj Reacta do swojej strony
 permalink: docs/add-react-to-a-website.html
 redirect_from:
   - "docs/add-react-to-an-existing-app.html"
 prev: getting-started.html
 next: create-a-new-react-app.html
 ---
 
-Use as little or as much React as you need.
+Używaj Reacta wybiórczo w miarę potrzeb.
```

Comment:
Proszę, wywal to zdanie :D Tak mnie razi, że za chwilę pojawia się jego kopia, że nie mogę na to patrzeć :P

Reply:
> Proszę, wywal to zdanie :D Tak mnie razi to, że za chwilę pojawia się jego kopia, że nie mogę na to patrzeć :P

Ale wtedy cały nagłówek i formatowanie się rozwali - masz jakieś propozycje jak tego uniknać?

Reply:
Ale wtedy cały nagłówek i formatowanie się rozwali - masz jakieś propozycje jak tego uniknać?